Cargo.toml

```diff
@@ -42,18 +42,18 @@ verify = ["ring"]
 validate = []
 
 [dependencies]
-asn1-rs = { version = "0.8.0", features=["datetime"] }
+asn1-rs = { version = "0.8.0", features=["bigint", "datetime"] }
 data-encoding = "2.2.1"
 lazy_static = "1.4"
 nom = "8.0"
-oid-registry = { version="0.8.1", features=["crypto", "x509", "x962"] }
+oid-registry = { version="0.9", features=["crypto", "x509", "x962"] }
 rusticata-macros = "5.0"
 ring = { version="0.17.12", optional=true }
-der-parser = { version = "10.0", features=["bigint"] }
+# der-parser = { version = "10.0", features=["bigint"] }
```

Comment:
It looks like the der-parser imports are all commented out instead of removed. I think it'd be sensible to do a pass across the branch to clean that stuff up before merge. I haven't flagged each instance individually in my review for that reason.

Reply:
I will add a patch to remove all remains (there are a few comments in files)